### DISCUSSION TITLE
How to make H3 heading visible in TOC

### DISCUSSION THREAD

**Discussion by mukhammart** [requarks/wiki#7422] [Help / Questions]
There is a question of how to make H3 heading visible in TOC. It now needs for better navigation, since the article can be too long and with at least H3 heading in TOC it can be easier to search for neccessary info.

**Comment by JamesEdmondsUKPN** [comment]
Did you ever get an answer to this one?
We are starting to put more stuff in our Wiki.js instance, but the apparent limitation of only H1 and H2 headings being picked up by the TOC is a bit of a pain.

**Comment by Mokkq** [comment]
Hi,

It's really annoying that it doesn't work. I built my own table of contents for this
by adding the code in the appendix under Admin > Overwrite Theme CSS
and adding the script to the HTML body. 

<img width="2299" height="478" alt="grafik" src="https://github.com/user-attachments/assets/70507c11-8c4b-4f4f-bf2c-bc3d45da9bc9" />

Both contents are in the appendix.
[html body.txt](https://github.com/user-attachments/files/23768227/html.body.txt)
[css overwrite.txt](https://github.com/user-attachments/files/23768228/css.overwrite.txt)
